README.md

```diff
@@ -1302,7 +1302,7 @@ arXiv, 2021.
 - <a name="Zhou21tro"></a>Zhou, Y., Gallego, G., Shen, S.,  
 *[Event-based Stereo Visual Odometry](https://doi.org/10.1109/TRO.2021.3062252)*,  
 IEEE Trans. Robot. (TRO), 2021. [Project page](https://sites.google.com/view/esvo-project-page/home), [PDF](https://arxiv.org/pdf/2007.15548), [YouTube](https://youtu.be/3CPPs1gz04k), [Code](https://github.com/HKUST-Aerial-Robotics/ESVO.git).
-
+- [Xiao et al., Arxiv](#Xiao22icrmv),
```

Comment:
The line after this one should be the title of the paper. I think this line is missing. Could you add it?

Reply:
I see the example of Bardow et al., CVPR 2016 and they did not add the title?

Reply:
Perhaps is not the best example because it appears multiple times (in dataset section). Let me explain. Bardow's paper is about optical flow estimation and image intensity reconstruction. The main entry is in the optical flow estimation section, and the entry in the image intensity reconstruction section is thus secondary (5th entry [here](https://github.com/uzh-rpg/event-based_vision_resources/blob/master/README.md#intensity-image-reconstruction-from-events)), linking to the main one. Could you revise your secondary entry? If not, I can do it.

Reply:
Ok, I have added.